pkg/skaffold/jib/jib.go

```diff
@@ -17,64 +17,163 @@ limitations under the License.
 package jib
 
 import (
+	"encoding/json"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"sort"
+	"strings"
+	"time"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
 	"github.com/karrick/godirwalk"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
-func getDependencies(cmd *exec.Cmd) ([]string, error) {
-	stdout, err := util.RunCmdOut(cmd)
-	if err != nil {
+// filesTemplate contains fields parsed from Jib's JSON output
+type filesTemplate struct {
```

Comment:
naming: why is this a Template?
how about
```suggestion
type pathMap struct {
```
?

Reply:
Ah this is a convention I took from Jib, where classes defining JSON templates are named with "template". I can change this.